cmd/swarmd/main.go

```diff
@@ -55,7 +55,6 @@ var (
 			}
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			ctx := context.Background()
```

Comment:
shouldn't use `return n.Err(ctx)` instead of `return n.Err(context.Background())` in https://github.com/docker/swarmkit/blob/master/cmd/swarmd/main.go#L177?